docs/releasing.md

```diff
@@ -86,3 +87,26 @@ CAPM3 and the released version should point to the release branch.
 If you just created a release branch (i.e. minor version release), you should
 modify the image for CAPM3 deployment in this branch to be tagged with the
 branch name. The image will then follow the branch.
+
+### Create a new tag for Ironic and BMO images
```

Comment:
Note to myself. What is the motivation behind to add a prefix to these container images? is there a specific case or reason etc.

Reply:
For every release of CAPM3 we are specifying the current version of Ironic and BMO images which work with the current CAPM3 code. Since at the time there is no images with the future tag, we are creating them at the same time.

Reply:
I see `capm3-v0.4.3` baremetal-operator container image in quay, and that is for CAPM3 v0.4.3 release as I got to know. That means we have to adapt m3-dev-env too to use those images accordingly?

Reply:
Probably yes